core/templates/pages/topic-editor-page/services/topic-editor-state.service.ts

```diff
@@ -198,6 +198,7 @@ angular.module('oppia').factory('TopicEditorStateService', [
             EditableTopicBackendApiService.fetchStories(topicId).then(
               function(canonicalStorySummaries) {
                 _setCanonicalStorySummaries(canonicalStorySummaries);
+                $rootScope.$apply();
```

Comment:
Can we use `$rootScrope.$applyAsync()` here and in other places? This helps avoid errors due to triggering a digest cycle while another digest cycle is in progress.

Reply:
Done